web/webkit/src/main/scala/net/liftweb/http/LiftRules.scala

```diff
@@ -647,10 +647,10 @@ class LiftRules() extends Factory with FormVendor with LazyLoggable {
    *
    * The HTML Lift emitted would then look like:
    *
-   * {{{
+   * <pre>
    * <span id="lift-event-js-F827001738725NKMEQNF"
    *       data-lift-removed-attributes="onclick">Do something!</span>
-   * }}}
+   * </pre>
```

Comment:
👎👎👎👎👎👎👎👎👎👎

Reply:
Yeah, I'm not wild about it either, but the file was (and remains) totally unusable in my tooling. I'm honestly not sure it shouldn't be given that the intent of this syntax is to include scala code, not HTML.

Reply:
Is that the case? Many uses of this include the scala REPL prompt, for example, which is not valid scala code. Additionally, the [scaladoc docs](https://docs.scala-lang.org/overviews/scaladoc/for-library-authors.html) just say ‘code blocks’, not ‘Scala code blocks’. It would seem surprising to me for scaladoc to have an overly narrow definition of code blocks, but not surprising for tooling to fail to take this flexibility into account, fwiw. Could def be wrong though.

Reply:
Hmmm. Yeah, guess I'm wrong on that then. Whatevs. I'll revert.